src/EnergyPlus/HVACDXHeatPumpSystem.cc

```diff
@@ -604,16 +598,16 @@ namespace HVACDXHeatPumpSystem {
         int I;                        // interation increment
         Real64 SpeedRatio;            // speed ratio between two neighboring speeds
 
-        auto &DXHeatPumpSystem(state.dataHVACDXHeatPumpSys->DXHeatPumpSystem);
+        auto &DXHeatPumpSystem = state.dataHVACDXHeatPumpSys->DXHeatPumpSystem(DXSystemNum);
```

Comment:
[src/EnergyPlus/HVACDXHeatPumpSystem.cc:607]:(style),[constVariable],Variable 'DXHeatPumpSystem' can be declared as reference to const